docker-compose.yml

```diff
@@ -47,13 +47,14 @@ services:
           nocopy: true
 
   db:
-    image: postgres:11.5
+    image: postgres:latest
```

Comment:
we do prefer strict version definition

Reply:
According to [postgres](https://www.postgresql.org/support/versioning/) the latest minor is recommended, which is equal to the `latest` tag regarding the latest major.